sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -44,18 +55,33 @@ export class ThrottlingRetryPolicy extends BaseRequestPolicy {
       if (isRestErrorWithHeaders(err)) {
         const delayInMs = getDelayInMs(err.response.headers);
 
-        if (delayInMs == null) {
+        if (
+          delayInMs == null ||
+          (this.retryOptions.maxRetryDelayInMs && delayInMs > this.retryOptions.maxRetryDelayInMs)
```

Comment:
should we just `delay(maxRetryDelayInMs)` for the case of `(this.retryOptions.maxRetryDelayInMs && delayInMs > this.retryOptions.maxRetryDelayInMs)`?

Reply:
Makes sense.

Reply:
Updated.